admin/js/admin.js

```diff
@@ -74,124 +74,128 @@
 
     $(function() {
 
-        //Show and hide one-time volunteer opportunity fields
-        $('#one-time-opportunity').change(function() {
-            if( this.checked ) {
-                $( '.one-time-field' ).show();
-                $( '.flexible-field' ).hide();
-            }
-            else {
-                $( '.one-time-field' ).hide();
-                $( '.flexible-field' ).show();
-            }
-        });
+        //Only run on WI Volunteer Management Opportunity New and Edit screens.
+        if( typeof pagenow != 'undefined' && pagenow == 'volunteer_opp' ) {
```

Comment:
Is this the only real change here? It's so hard to tell what else you adjusted given the spacing changes below.

Reply:
Yeah that is the only change, I'm not sure why the formatting had changed so much - I just  checked out the files and made this one update.